app/src/main/res/values/styles.xml

```diff
@@ -202,6 +210,14 @@
         <item name="lineHeight">28sp</item>
     </style>
 
+    <style name="P.DialogTitle">
```

Comment:
This style seems to be identical to `H2`. Do we need it?

Reply:
No and removed.